### PR TITLE
Create temporary Updates page for Pocket

### PR DESCRIPTION
**Pull request by reemhamz**
## One-line summary
The Pocket team wants to launch an email updates page, and this is a temporary hold with Pocket's press email.

🚨Don't merge just yet 🚨

## Issue / Bugzilla link
#13123 

## Testing

To test this work:

- [ ] http://localhost:8000/en/updates-signup/
- [ ] https://www-demo1.tekcopteg.com/updates-signup/

### CODE REVIEW

File: bedrock/pocket/templates/pocket/updates-signup.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  {% call split(
+    block_class='mzp-l-split-center-on-sm-md',
+    media_class='mzp-l-split-h-center mzp-l-split-media-overflow',
+    media_after=True,
+    image=resp_img(
+      url='img/pocket/pocket-update-signup.png',
+      optional_attributes={
+        'class': 'mzp-c-split-media-asset'
+      }
+      ),
```
Comment (nathan-barrett) [inline bedrock/pocket/templates/pocket/updates-signup.html:34]: Nit: Indentation

**Review by nathan-barrett — APPROVED** [review]
r+wc just needs the redirect and URL updates that Dan mentioned in the issue before it can get merged. Approving so you don't need to wait for my re-review on monday.

File: bedrock/pocket/redirects.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from bedrock.redirects.util import redirect
+
+redirectpatterns = (redirect(r"^pocket-updates-signup/?$", "pocket.updates-signup"),)
```
Comment (reemhamz) [inline bedrock/pocket/redirects.py:7]: Waiting for @stevejalim to get help setting up the Pocket backend to handle redirects

**Comment by stevejalim** [comment]
FYI @reemhamz @nathan-barrett Am working on supporting redirects in Pocket Mode, because that's basically new behaviour for Pocket Mode (we disabled redirects when splitting things up, and now need to support a version of it again). On it!

**Comment by reemhamz** [comment]
Thanks @stevejalim! Cc: @slightlyoffbeat for visibility

**Comment by reemhamz** [comment]
Thanks @stevejalim for updating the branch!

**Comment by stevejalim** [comment]
Related PR for changes to the Pocket gateway https://github.com/Pocket/dotcom-gateway/pull/112

**Pull request merged**